src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -167,6 +167,18 @@ public static Calendar convertToUserTimeZone(Calendar time, double timeZone) {
     }
 
     /**
+     * Converts the {@code localDate} to UTC by adding/subtracting the {@code localTimeZone} offset.
```

Comment:
The `adding/subtracting` part may be confusing and is an implementation detail. Suggestion:
```
Converts the {@code localDate} (currently in the {@code localTimeZone}) to UTC through shifting by the offset.
```